package.json

```diff
@@ -10,6 +10,7 @@
     "@types/numeral": "^0.0.28",
     "@types/react-animate-on-scroll": "^2.1.2",
     "@types/react-copy-to-clipboard": "^4.3.0",
+    "@types/react-js-pagination": "^3.0.3",
```

Comment:
Is there any need for this package? I don't see it used anywhere.

Reply:
I can't see any good reason, happy you removed. 👍